test/client/fixtures/sandbox/console-test.js

```diff
@@ -47,4 +47,16 @@ if (window.console && typeof window.console.log !== 'undefined') {
             info:  originMethods.info
         };
     });
+
+    test('consoleMethCalled event after document.write', function () {
```

Comment:
`'consoleMethod' event should be raised after document.write`

Reply:
'consoleMethCalled' event...

Reply:
Yep